tianshou/utils/net/discrete.py

```diff
@@ -68,16 +64,22 @@ def conv2d_layers_size_out(size,
         convw = conv2d_layers_size_out(w)
         convh = conv2d_layers_size_out(h)
         linear_input_size = convw * convh * 64
-        self.fc = nn.Linear(linear_input_size, 512)
-        self.head = nn.Linear(512, action_shape)
+
+        self.net = nn.Sequential(
+            nn.Conv2d(4, 32, kernel_size=8, stride=4),
+            nn.ReLU(inplace=True),
+            nn.Conv2d(32, 64, kernel_size=4, stride=2),
+            nn.ReLU(inplace=True),
+            nn.Conv2d(64, 64, kernel_size=3, stride=1),
+            nn.ReLU(inplace=True),
+            nn.Flatten(),
```

Comment:
Flatten(1)

Reply:
``nn.Flatten()`` means ``x.flatten(1)`` by default. Check the [doc](https://pytorch.org/docs/1.4.0/nn.html#torch.nn.Flatten) here.